{
  logger.error("An uncaught exception occurred during HTTP invocation",throwable);
  return super.toResponse(throwable);
}
